docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<p>Do you think we should provide this explanation in the beginning of this tutorial or not? Because, I think this explains why one should think about / consider discovering TSRC in this data. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>